php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -556,7 +556,9 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                             'instrument_title',
                             $pieces[1]
                         );
-                        $this->_addMetadataFields();
+                        if ($this->DataEntryType!=="DirectEntry") {
```

Comment:
from what i can tell, the DataEntryType is never set for surveys (display method doesn't get called) so changing this check to `$this->DataEntryType === 'normal'` would fix this check